style.css

```diff
@@ -0,0 +1,914 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+ul {
+  list-style: none;
+}
+
+/* ========== mobile-veiw =============== */
+
+/* =========== first section Home page ======== */
+
+.main-section {
+  background-image: url(img/mobile_intro_bg.png);
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  margin-bottom: 3rem;
+}
+
+.main-section-article {
+  margin-top: 4rem;
+}
+
+article h2 {
+  font-size: x-large;
+  font-weight: 900;
+  text-transform: uppercase;
+}
+
+article h1,
+article h2 {
+  color: #ec5342;
+}
+
+.main-section-article h1,
+.main-section-article h2 {
+  width: 90%;
+  text-align: left;
+  padding-bottom: 1rem;
+  margin: auto 1rem;
+}
+
+.main-section-article h2 {
+  margin-bottom: 2rem;
+}
+
+.article-1 {
+  color: #272a31;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  margin-left: 2rem;
+  background-color: white;
+  width: 80%;
+  height: 38vh;
+  text-align: left;
+}
+
+#hamburger {
+  position: fixed;
+  z-index: 999;
+  cursor: pointer;
+  color: #262930;
+  font-size: 40px;
+  margin-top: 1rem;
+  width: 100%;
+}
+
+.menu-container {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  position: fixed;
+  z-index: 999;
+  width: 100%;
+  background-color: white;
+  margin-left: -1rem;
+  margin-top: -1rem;
+}
+
+.menu-container img {
+  display: block;
+}
+
+.date {
+  color: #272a31d3;
+  font-size: x-large;
+  font-weight: 700;
+  width: 100%;
+  margin-bottom: 0.5rem;
+  margin-left: 1rem;
+}
+
+.place {
+  color: #272a31;
+  font-size: small;
+  font-weight: 400;
+  width: 100%;
+  margin-left: 1rem;
+}
+
+/* ======== second section  Home Page =============== */
+
+.second-section {
+  background-color: #262930;
+  background-image: url(/img/pattern_bg.png);
+  background-size: auto;
+  background-repeat: repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 1rem;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1rem;
+}
+
+.second-section h2 {
+  color: aliceblue;
+  margin-top: 1rem;
+  margin-bottom: 0;
+  width: 70%;
+  text-align: center;
+}
+
+.hr-about-1 {
+  color: #ec5342;
+  width: 9%;
+  margin-top: -0.5rem;
+  margin-bottom: 1rem;
+}
+
+.second-section .card {
+  border: 1px solid blue;
+  background-color: #3c3f46;
+  opacity: 0.7;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  text-align: center;
+  padding: 1rem;
+  width: 90%;
```

Comment:
- [ ] You still have not made this section content align properly. Please refer to previous reviewers [images here](https://github.com/MostafaShahat2021/capstone-project---Conference-page-Module-1/pull/2#discussion_r978959351). I suggest you remove the CSS style of `margin-left: 5rem;` from this element. 

```
.card .content {
    color: aliceblue;
    width: 70%;
    text-align: left;
    margin-left: 5rem; 
}
```
- [ ] 
- [ ] And it currently has blues border on it. please remove this as this is not part of the design.